src/games/strategy/triplea/settings/battle/options/BattleOptionsSettings.java

```diff
@@ -3,10 +3,11 @@
 import games.strategy.triplea.settings.HasDefaults;
 import games.strategy.triplea.settings.SystemPreferenceKey;
 import games.strategy.triplea.settings.SystemPreferences;
-import games.strategy.triplea.ui.BattleDisplay;
+import games.strategy.triplea.ui.battledisplay.BattleDisplay;
 
 public class BattleOptionsSettings implements HasDefaults {
 
+  static final int DEFAULT_BATTLE_DICE_PER_ROW = 18;
```

Comment:
- by using vertical space there should be scrollbars less of the time now as compared to before when the dice were always on one line

- the 'optimal' count would depend on the size of the battle window, not screen. It would need to be updated whenever the screen was resized, and the display redrawn. (note: the battle window size is actually hardcoded on window launch)

So I mostly agree, but dynamically calculating the value would be a much more tricky/complicated change. This is intended to be a relatively simple update for the moment.